### PR TITLE
Docs directory restructure: Partials, images, third-party docs

### PR DESCRIPTION
**Pull request by symbolpunk**
- This PR is a continuation of our docs file structure work, focusing on partials, images, and other finishing touches.
 - Context and process notes are available [here](https://www.notion.so/arbitrum/TW-Sprint-Doc-April-60220d7b7539494eb2c69fbbd92081df?pvs=4#684ea1095df8489c835885c0a3be969e)
 - Previous batches: https://github.com/OffchainLabs/arbitrum-docs/pull/1115 | https://github.com/OffchainLabs/arbitrum-docs/pull/1199

Note that some file relocations in this PR do **_not_** include the addition of a number prefix - intent is to address those types of non-blocking todos in a separate PR.

TODOs before merging:

- [x] test `yarn update:all`
- [x] test 3p docs catch-all redirect

### CODE REVIEW

File: arbitrum-docs/partials/_troubleshooting-orbit-partial.md [hunk 1]
```diff
@@ -1,198 +1,106 @@
...
-
-<p>
-  Yes! Arbitrum Orbit's core technology has undergone a comprehensive audit and is now able to
-  support deployments to mainnet. You can read more about it{' '}
-  <a href="https://docs.arbitrum.io/launch-orbit-chain/concepts/public-preview-expectations#arbitrum-orbit-is-mainnet-ready-but-deploy-to-testnet-first">
-    here
-  </a>
-  .
-</p>
+<p>Yes! Arbitrum Orbit's core technology has undergone a comprehensive audit and is now able to support deployments to mainnet. You can read more about it <a href="https://docs.arbitrum.io/launch-orbit-chain/concepts/public-preview-expectations#arbitrum-orbit-is-mainnet-ready-but-deploy-to-testnet-first">here</a>.</p>
```
Comment (TucksonDev) [inline arbitrum-docs/partials/_troubleshooting-orbit-partial.md:2]: Can you run `yarn format` one more time to remove these changes?

File: vercel.json [hunk 2]
```diff
@@ -1,5 +1,55 @@
 {
     "redirects": [
+        {
+            "source": "/(intro)",
+            "destination": "/welcome/arbitrum-gentle-introduction",
+            "permanent": false
+        },
+        {
+            "source": "/for-devs/third-party-docs/:slug*",
```
[the remark this answers is not in the document]
Comment (TucksonDev) [reply vercel.json:9]: Oh, that's very nice! 🔥
Is it possible to add the `/?` at the end? It's not critical, imo, but the current links have a `/` at the end, and fail to redirect if we add that (e.g. `https://arbitrum-docs-git-file-restructure-6-offchain-labs.vercel.app/for-devs/third-party-docs/Envio/`).

**Review by TucksonDev — COMMENTED** [review]
Amazing job, sir! 🚀 
I've left 2 minor comments, but more importantly, can you merge master into it and resolve the conflicts? `master` has now docusaurus 3, which I suppose doesn't really affect these changes, but I'd like to take a final look with that branch merged 🙏

File: vercel.json (hunk 2 above)
Comment (anegg0) [inline vercel.json:9]:
```suggestion
            "source": "/for-devs/third-party-docs/:slug*/?",
```

Comment (anegg0) [reply vercel.json:9]: Is this the modification you required, @TucksonDev?

**Review by anegg0 — APPROVED** [review]
@TucksonDev, picking this up, I added the modification you requested (hopefully, I got it right!).

**Comment by anegg0** [comment]
This PR has been stale for too long. Closing it now.

**Pull request closed without merging**